hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/FutureMessage.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+
+/**
```

Comment:
well... not that easy... there is more than the message state that needs to go into the future.  the full class needs to be maintained.  I will assume inner class again.